### DISCUSSION TITLE
Can loco app be hosted under apache server ?

### DISCUSSION THREAD

**Discussion by tjyang** [loco-rs/loco#280] [Q&A]
Can "myapp-cli start" using apache server on the  OS ?

**Comment by jondot** [comment]
I believe nginx will be a better choice. For Apache, I would start here https://httpd.apache.org/docs/2.4/howto/reverse_proxy.html

**Reply by LimpidCrypto** [reply]
Maybe this can help you as well:

https://github.com/loco-rs/loco/discussions/279

**Comment by tjyang** [comment]
@LimpidCrypto , Thanks for the reply about your plesk how-to. Looks to me it is still running docker container with its own httpd service.
Hoping to have "myapp-cli start"  to serve loco's http traffic under apache web server, ie disable loco's own web server.
There are still many  production apache web servers across intranet/internet.
